prime-dotnet-webapi/Services/AutomaticAdjudicationService.cs

```diff
@@ -134,38 +134,41 @@ protected override async Task<bool> ProcessRuleInternal(Enrollee enrollee)
 
                 foreach (var cert in enrollee.Certifications)
                 {
-                    PharmanetCollegeRecord record = null;
-                    try
+                    if (cert.License.Validate)
```

Comment:
Could you please flip the logic and make this
`if(!cert.Licencse.Validate)
{
    continue;
}`
it makes it easier to read in my opinion

Reply:
or, alternatively, you could use
`foreach (var cert in enrollee.Certifications.Where(c => c.License.Validate))`

Reply:
> or, alternatively, you could use
> `foreach (var cert in enrollee.Certifications.Where(c => c.License.Validate))`

I like this better the other ways feels awkward to me